### ISSUE TITLE
UNUSED_LOCAL_VARIABLE false positive when used in new object initialization (2)

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#640]
Follow-up for #577 

```cfml
component {
    /**
     * Some function hint
     */
    public User function createUser()
    {
        var userName = "KamasamaK";
        var user = new User(userName);

        return user;
    }
}
```

In the code above, `userName` erroneously triggers the UNUSED_LOCAL_VARIABLE rule.

**Comment by ryaneberly** [comment]
hmm. works for me on dev

**Comment by KamasamaK** [comment]
I just tested against the latest build and this still isn't fixed. I see there was [a test](https://github.com/cflint/CFLint/blob/5e6275ca6fd69a8f83f05bff4910321b92190ab7/src/test/resources/com/cflint/tests/UnusedVariable/newobj_640.cfc) added, but it is not formatted as a component. I will try to format my code snippets as entire components in the future.

**Comment by ryaneberly** [comment]
fixed in dev :)
